pallets/common/src/lib.rs

```diff
@@ -14,8 +14,45 @@
 // You should have received a copy of the GNU General Public License
 // along with Unique Network. If not, see <http://www.gnu.org/licenses/>.
 
+//! # Common pallet
+//!
+//! The Common pallet provides functionality for handling collections.
+//!
+//! ## Overview
+//!
+//! The Common pallet provides functions for:
+//!
+//! - Setting and approving collection soponsor.
+//! - Get\set\delete allow list.
+//! - Get\set\delete collection properties.
+//! - Get\set\delete collection property permissions.
+//! - Get\set\delete token property permissions.
+//! - Get\set\delete collection administrators.
+//! - Checking access permissions.
+//! - Provides an interface for common collection operations for different collection types.
+//! - Provides dispatching for implementations of common collection operations, see [dispatch] module.
+//! - Provides functionality of collection into evm, see [erc] and [eth] module.
+//!
+//! ### Terminology
+//! **Collection sponsor** - For the collection, you can set a sponsor, at whose expense it will
+//! be possible to mint tokens.
+//!
+//! **Allow list** - List of users who have the right to minting tokens.
+//!
+//! **Collection properties** - Collection properties are simply key-value stores where various
+//! metadata can be placed.
+//!
+//! **Collection property permissions** - For each property in the collection can be set permission
```

Comment:
Buuut we don't though? There are no permissions for collection properties, only for tokens.